docs/operator-manual/user-management/index.md

```diff
@@ -508,6 +513,55 @@ data:
   ...
 ```
 
+#### Store full OIDC config in secret
+
+If you want to store the entirety of the OIDC config in a secret you may do so by
+referencing the secret and key in that secret in same manner as noted for the `clientID`
+and `clientSecret` above.
+
+Syntax: `$<k8s_secret_name>:<a_key_in_that_k8s_secret>`
+
+> NOTE: Secret must have label `app.kubernetes.io/part-of: argocd`
+
+##### Example
+
+`another-secret`:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: another-secret
+  namespace: argocd
+  labels:
+    app.kubernetes.io/part-of: argocd
+type: Opaque
+stringData:
+  ...
+  # Store client secret like below.
+  # If stringData is used non-base-64 encoded data can be provided
+  oidc.config: |
+    name: Auth0
+    clientSecret: xxxxxxxxx
+    clientID: aabbccddeeff00112233
+  ...
+```
+
+`argocd-cm`:
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-cm
+  namespace: argocd
+  labels:
+    app.kubernetes.io/name: argocd-cm
+    app.kubernetes.io/part-of: argocd
+data:
+  ...
+  oidc.config: $another-secret:oidc.auth0
```

Comment:
hi @jskrzypek 
should this say `$anoter-secret:oidc.config` instead?

Reply:
Good catch, thanks!